### DISCUSSION TITLE
Automatic attachiment of incoming form to dropdown animal - and if can be submitted to animal diary.

### DISCUSSION THREAD

**Discussion by terrytaylor01** [sheltermanager/asm3#1560] [Q&A / Help]
Hi, just facing an issue with how our rescue is handling our animal check in's from foster carers. We were looking into the form documentation and see how it can be attached to an animal via an `animalname` field.

However as the idea is that these forms would come in and be automatically attached to the animal that was selected in in animalname dropdown. Similarly the current way check in's is handled in the rescue is using the diary as it seems the most efficient in parsing simple text - testing currently we can only get the form submissions as media. 

Would appreciate any guidance on these topics, thank you!

**Comment by Tetzor** [comment]
Hi, sorry I'm not very clear on what the problem is, what is it that you are specifically trying to do?
